docs/kqueen.cli_gcloud_examples.rst

```diff
@@ -0,0 +1,150 @@
+
+https://cloud.google.com/sdk/gcloud/reference/init
+
+#. Login to Google Cloud:
+
+.. code-block:: bash
+
+   $ cd /tmp/
+   $ gcloud init
+
+#. Create new project in GCE:
+
+.. code-block:: bash
+
+   $ GCE_PROJECT_NAME="My Test Project"
+   $ GCE_PROJECT_ID="my-test-project-`date +%F`"
+   $ gcloud projects create "$GCE_PROJECT_ID" --name "$GCE_PROJECT_NAME"
+   Create in progress for [https://cloudresourcemanager.googleapis.com/v1/projects/my-test-project-2018-05-11].
+   Waiting for [operations/cp.7874714971502871878] to finish...done.
+
+#. Set new project as default:
+
+.. code-block:: bash
+
+   $ gcloud config set project $GCE_PROJECT_ID
+
+#. Enable Billing for the project:
+
+   $ GCE_ACCOUNT_ID=$(gcloud beta billing accounts list --filter="My Billing Account" --format="value(ACCOUNT_ID)")
+   $ gcloud billing projects link "$GCE_PROJECT_ID" --billing-account="$GCE_ACCOUNT_ID"
+
+#. Create Service Account and assign it proper rights:
+
+.. code-block:: bash
+
+   $ GCE_SERVICE_ACCOUNT_DISPLAY_NAME="My_Service_Account"
+   $ GCE_SERVICE_ACCOUNT_NAME="my-service-account"
+   $ gcloud iam service-accounts create "$GCE_SERVICE_ACCOUNT_NAME" --display-name "$GCE_SERVICE_ACCOUNT_DISPLAY_NAME"
+
+#. Generate keys for new service account:
+
+.. note:: Keep the generated key.json file in a safe location.
+
+.. code-block:: bash
+
+   $ GCE_SERVICE_ACCOUNT_EMAIL=$(gcloud iam service-accounts list --filter="$GCE_SERVICE_ACCOUNT_NAME" --format="value(email)")
+   $ gcloud iam service-accounts keys create --iam-account $GCE_SERVICE_ACCOUNT_EMAIL key.json
+
+#. Assign a role to the service account and bind it to the project:
+
+.. code-block:: bash
+
+   $ GCE_EMAIL="your_email_address@gmail.com"
+   $ gcloud iam service-accounts add-iam-policy-binding "$GCE_SERVICE_ACCOUNT_EMAIL" --member="user:$GCE_EMAIL" --role="roles/owner"
+   $ gcloud projects add-iam-policy-binding $GCE_PROJECT_ID --member="serviceAccount:$GCE_SERVICE_ACCOUNT_EMAIL" --role="roles/container.clusterAdmin"
+   $ gcloud projects add-iam-policy-binding $GCE_PROJECT_ID --member="serviceAccount:$GCE_SERVICE_ACCOUNT_EMAIL" --role="roles/iam.serviceAccountActor"
+
+To create / configure the GKE cluster using KQueen follow these steps:
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+#. Start KQueen and obtain bearer token:
+
+.. code-block:: bash
+
+   $ git clone https://github.com/Mirantis/kqueen.git
+   $ cd kqueen
+   $ docker-compose -f docker-compose.yml -f docker-compose.demo.yml rm -f # Make sure you are starting from scratch
+   $ docker-compose -f docker-compose.yml -f docker-compose.demo.yml up
+   $ TOKEN=$(curl -s -H "Content-Type: application/json" --data '{ "username": "admin", "password": "default" }' -X POST 127.0.0.1:5000/api/v1/auth | jq -r '.access_token')
+   $ echo $TOKEN
+
+#. Create new organization "testorganization" with new user/password "testusername / testpassword":
```

Comment:
Why do you sometimes use "a/b" and sometimes "a / b"? Maybe consistency would be better